community/examples/intel/hpc-slurm-daos.yaml

```diff
@@ -114,61 +114,68 @@ deployment_groups:
         destination: /tmp/daos_client_config.sh
 
   - id: debug_nodeset
-    source: community/modules/compute/schedmd-slurm-gcp-v6-nodeset
+    source: github.com/GoogleCloudPlatform/hpc-toolkit//community/modules/compute/schedmd-slurm-gcp-v6-nodeset?ref=v1.32.0&depth=1
     use: [network1]
     settings:
       name: ns1
       node_count_dynamic_max: 4
       machine_type: n2-standard-2
       enable_placement: false  # the default is: true
-      service_account_scopes:
-      - "https://www.googleapis.com/auth/monitoring.write"
-      - "https://www.googleapis.com/auth/logging.write"
-      - "https://www.googleapis.com/auth/devstorage.read_only"
-      - "https://www.googleapis.com/auth/cloud-platform"
+      service_account:
```

Comment:
In theory it should have worked with 1.33 which would have avoided some of the interface changes (like service account). What was the issue with 1.33.

Reply:
Theory is correct.